docs/Concepts/Privacy/PrivateTransactionManager.md

```diff
@@ -11,52 +11,158 @@ The Private Transaction Manager has two distinct components:
+        timeout = 5
+        httpIdleConnTimeout = 10
+        httpWriteBufferSize = 4096
+        httpReadBufferSize = 4096
+        ```
+
+        Where:
+
+        * `tlsRootCA` is any combination of comma separated files/directories containing root CA certificates, defaults to host's certificates
+        * `tlsClientCert` is file containing client certificate
+        * `tlsClientKey` is file containing client certificate private key
+
+## Using command line parameters
+
+The command line parameters listed below can be used to configure the connection to the Transaction Manager.
+Note that these can be used in conjunction with the other options given above,
+in which case the command line parameters will override any others.
+
+```text
+QUORUM PRIVATE TRANSACTION MANAGER OPTIONS:
+  --ptm.socket value                  Path to the ipc file when using unix domain socket for the private transaction manager connection
+  --ptm.url value                     URL when using http connection to private transaction manager
+  --ptm.timeout value                 Timeout (seconds) for the private transaction manager connection. Zero value means timeout disabled. (default: 0)
+  --ptm.dialtimeout value             Dial timeout (seconds) for the private transaction manager connection. Zero value means timeout disabled. (default: 0)
+  --ptm.http.idletimeout value        Idle timeout (seconds) for the private transaction manager connection. Zero value means timeout disabled. (default: 0)
+  --ptm.http.writebuffersize value    Size of the write buffer (bytes) for the private transaction manager connection. Zero value uses http.Transport default. (default: 0)
+  --ptm.http.readbuffersize value     Size of the read buffer (bytes) for the private transaction manager connection. Zero value uses http.Transport default. (default: 0)
+  --ptm.tls.mode value                If "off" then TLS disabled (default). If "strict" then will use TLS for http connection to private transaction manager
+  --ptm.tls.rootca value              Path to file containing root CA certificate for TLS connection to private transaction manager (defaults to host's certificates)
+  --ptm.tls.clientcert value          Path to file containing client certificate (or chain of certs) for TLS connection to private transaction manager
+  --ptm.tls.clientkey value           Path to file containing client's private key for TLS connection to private transaction manager
+  --ptm.tls.insecureskipverify        Disable verification of server's TLS certificate on connection to private transaction manager
+```
+
+=== "Unix IPC socket connection"
+
+    Specify the path to the IPC socket file:
+
+    !!! example "Example IPC connection"
+
+        ```bash
+        geth <other parameters> \
+             --ptm.socket qdata/c1/tm.ipc
+        ```
+
+=== "HTTP connection"
+
+    This should only be used for development purposes, due to a lack of security on the connection.
```

Comment:
maybe tu this warning in a `!!! warning` box

Reply:
done